File: Drivo.Entities/DrivingEntity.cs
namespace Drivo.Entities
{
    public class DrivingEntity : EventEntity
    {
        public virtual StudentEntity? Student { get; set; }
        public int? StudentId { get; set; }
        public virtual InstructorEntity? Instructor { get; set; }
        public int? InstructorId { get; set; }
    }
}

File: Drivo/src/entities/DrivingEntity.ts
import { EventEntity } from "./event.entity"
import { InstructorEntity } from "./InstructorEntity";
import { StudentEntity } from "./StudentEntity";

export class DrivingEntity extends EventEntity {
    student: StudentEntity | null;
    studentId: number | null;
    instructor: InstructorEntity | null;
    instructorId: number | null;
}